homework/CSharpAdvanced/RepaintingFlowers.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace CSharpAdvanced
+{
+    public class RepaintingFlowers<T> where T : Flower
+    {
```

Comment:
Please rewrite the method here :)
You can leave GetNewPrice() method as it is, and in RepaintingFlowers reuse it. If new color != previous color - then repaint and increase price  :)